grim.go

```diff
@@ -146,6 +151,17 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	if len(url) >= 12 {
+		lastCommitNum := url[len(url)-12:]
+		if lastCommitNum == "000000000000" { //if last 12 digits is zero, it's a deleted branch
+			return false
+		}
+	}
+	return true //default is true
```

Comment:
Any time you make a comment that is the same as the code you should remove it.  Anyone who can read golang (ie anyone) understands that the default is true in this case.